recipe/meta.yaml

```diff
@@ -1,31 +1,36 @@
 {% set name = "skein" %}
-{% set version = "0.8.0" %}
+{% set version = "0.8.1" %}
+{% set pkg_loc = "py2.py3" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://files.pythonhosted.org/packages/py3/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}-py3-none-any.whl
-  sha256: 71da4a16e2bba4d99fa90454b58e951463452596a6e6afaaf024a0d1cce87d3f
+  url: https://files.pythonhosted.org/packages/{{ pkg_loc }}/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}-{{ pkg_loc }}-none-any.whl
+  sha256: 506884a13675ec884b1bbd85675f196fe123e13f33f6b308303531cd26466726
 
 build:
-  number: 1
-  skip: True  # [win or py<35]
-  script: python -m pip install --no-deps --ignore-installed {{ name }}-{{ version }}-py3-none-any.whl
+  number: 0
+  noarch: python
+  script:
+    - python -m pip install --no-deps --ignore-installed {{ name }}-{{ version }}-{{ pkg_loc }}-none-any.whl
+    - cp {{ SP_DIR }}/{{ name }}-{{ version }}.dist-info/LICENSE {{ SRC_DIR }}/LICENSE
```

Comment:
This is a little hacky. However as installing the `whl` does place the license file in `site-packages`, this is a reliable way to include it in the Conda package without extracting the `whl` manually and rummaging around for this file.

Reply:
Seems fine to me.